app/views/index.erb

```diff
@@ -0,0 +1,9 @@
+<h2> Current messages </h2>
+<ul>
+<% @peeps.reverse.each do |peep| %>
```

Comment:
You shouldn't rely on `@peeps.reverse` to create reverse chronological order.  Is there a way to push this logic closer to the database?